src/AzureStack/Subscriptions/Subscriptions/Microsoft.AzureStack.Management.Subscriptions.csproj

```diff
@@ -0,0 +1,19 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.AzureStack.Management.Subscriptions</PackageId>
+    <Description>Provides developers with libraries for the subscriptions under AzureStack to manage subscriptions</Description>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.AzureStack.Management.Subscriptions</AssemblyName>
+    <PackageTags>Microsoft Azure Stack;Subscription;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+        This is a preview release of the AzureStack Subscription Admin SDK.
```

Comment:
is this subscription admin SDK or management SDK?

Reply:
This is a user facing subscriptions SDK not admin, For admin, I have this PR https://github.com/Azure/azure-sdk-for-net/pull/4260